bench/metrics/mandelbrot.rb

```diff
@@ -105,6 +105,6 @@ def mandelbrot(size)
   loop do
     mandelbrot 750
   end
-rescue RubyTruffleError
+rescue RuntimeError
   puts Time.now - start
```

Comment:
This will catch possibly other errors in the benchmark, that's not ideal. Checking the exception's message would help.